src/main/java/com/textmessenger/controller/UserController.java

```diff
@@ -38,6 +38,13 @@ public UserController(UserService userService) {
             .orElse(ResponseEntity.notFound().build());
   }
 
+  @PostMapping("/find")
+  public ResponseEntity<?> findAllUsers(@RequestBody String str) {
+    System.out.println(str);
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: Replace this use of System.out or System.err by a logger. ([squid:S106](https://sonarcloud.io/organizations/final_project_text_messenger/rules#rule_key=squid%3AS106))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=Text_Messenger_Project&pullRequest=121&issues=AWU4Qf8vW2AOP0CkHewD&open=AWU4Qf8vW2AOP0CkHewD)

[ISSUE_KEY]: AWU4Qf8vW2AOP0CkHewD